packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -29,6 +29,11 @@ export const ENABLE_DIFF_NO_FAIL_CONTEXT = 'aws-cdk:enableDiffNoFail';
 /** @deprecated use `ENABLE_DIFF_NO_FAIL_CONTEXT` */
 export const ENABLE_DIFF_NO_FAIL = ENABLE_DIFF_NO_FAIL_CONTEXT;
 
+/**
+ * Switch to convention mode deployments which enable CI/CD
+ */
+export const CONVENTION_MODE_DEPLOYMENTS_CONTEXT = 'aws-cdk:conventionModeDeployments';
```

Comment:
This should begin with `@aws-cdk/core` (to denote the module that consumes it) and also the "convention mode" terminology is gone. How about `@aws-cdk/core:newDeploymentConfiguration` (or `newDeploymentProvider` or `newBootstrap`). I know, lame...

```suggestion
export const CONVENTION_MODE_DEPLOYMENTS_CONTEXT = '@aws-cdk/core:newDeploymentConfiguration';
```

Reply:
How about throwing something like "parameterless assets" in there?